sdks/nodejs/package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@google-cloud/agones-sdk",
-  "version": "1.3.0",
```

Comment:
I don't think this file should be part of the review. Please can it be reverted although we should also look at why it is out of date

Reply:
Thank you for checking!
It was a mistake...
revert and committed.